auto3dseg/algorithm_templates/dints/scripts/dummy_training_script.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+import nvidia_smi
+import numpy as np
+import os
+import torch
+import sys
+import yaml
+
+from monai.bundle import ConfigParser
+from monai.inferers import sliding_window_inference
+from torch.cuda.amp import GradScaler, autocast
+
+
+config_file = []
+config_file.append(
+    os.path.join("work_dir", "dints_0", "configs", "hyper_parameters.yaml")
+)
+config_file.append(os.path.join("work_dir", "dints_0", "configs", "network.yaml"))
+config_file.append(
+    os.path.join("work_dir", "dints_0", "configs", "transforms_train.yaml")
+)
+config_file.append(
+    os.path.join("work_dir", "dints_0", "configs", "transforms_validate.yaml")
+)
+
+parser = ConfigParser()
+parser.read_config(config_file)
+
+device = torch.device("cuda:0")
+torch.cuda.set_device(device)
+
+input_channels = parser.get_parsed_content("training#input_channels")
+output_classes = parser.get_parsed_content("training#output_classes")
+patch_size = parser.get_parsed_content("training#patch_size")
+patch_size_valid = parser.get_parsed_content("training#patch_size_valid")
+overlap_ratio = parser.get_parsed_content("training#overlap_ratio")
+
+model = parser.get_parsed_content("training_network#network")
+model = model.to(device)
+
+loss_function = parser.get_parsed_content("training#loss")
+optimizer_part = parser.get_parsed_content("training#optimizer", instantiate=False)
+optimizer = optimizer_part.instantiate(params=model.parameters())
+
+train_transforms = parser.get_parsed_content("transforms_train")
+
+data_stats_file = os.path.join("work_dir", "datastats.yaml")
+with open(data_stats_file) as f_data_stat:
+    data_stat = yaml.full_load(f_data_stat)
+
+pixdim = parser.get_parsed_content("transforms_train#transforms#3#pixdim")
+pixdim = [np.abs(pixdim[_i]) for _i in range(3)]
+
+max_shape = [-1, -1, -1]
+for _k in range(len(data_stat["stats_by_cases"])):
+    image_shape = data_stat["stats_by_cases"][_k]["image_stats"]["shape"][0]
+    image_spacing = data_stat["stats_by_cases"][_k]["image_stats"]["spacing"][0]
+    image_spacing = [np.abs(image_spacing[_i]) for _i in range(3)]
```

Comment:
I am using the data analyzer from the MONAI Core, which I think average all channels by default. So the`[0]` may be redundant and I ran into errors when I run the code as below: 

```
(base) root@adminn-MS-7D31:/workspace/monai/research-contrib-in-dev# python work_dir/dints_0/scripts/dummy_training_script.py
Traceback (most recent call last):
  File "work_dir/dints_0/scripts/dummy_training_script.py", line 59, in <module>
    image_spacing = [np.abs(image_spacing[_i]) for _i in range(3)]
  File "work_dir/dints_0/scripts/dummy_training_script.py", line 59, in <listcomp>
    image_spacing = [np.abs(image_spacing[_i]) for _i in range(3)]
TypeError: 'float' object is not subscriptable
```